resources/js/Pages/Admin/Rooms/CreateRoomForm.vue

```diff
@@ -45,12 +76,75 @@
                         <jet-input-error :message="createRoomForm.error('status')" class="mt-2"/>
                     </div>
 
-                    <div class="col-span-6 sm:col-span-6">
+                    <div class="col-span-6 sm:col-span-3">  
+                        <jet-label for="equippment" value="Equippment"/>
+                        <br>
+                        <label class="flex items-center">
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.projector"/>
+                        <span class="ml-2 text-sm text-black">Projector</span>    
+                        </label> 
+                        <label class="flex items-center">
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.television"/>
+                        <span class="ml-2 text-sm text-black">Television</span>  
+                        </label> 
+                        <label class="flex items-center">
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.computer"/>
+                        <span class="ml-2 text-sm text-black">Computer</span>
+                        </label>   
+                        <label class="flex items-center"> 
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.whiteboard"/>
+                        <span class="ml-2 text-sm text-black">Whiteboard</span>    
+                        </label>                 
+                        <jet-input-error :message="createRoomForm.error('equippment')" class="mt-2"/>
+                    </div>
+
+                    <div class="col-span-6 sm:col-span-3">  
+                        <jet-label for="restrictions" value="Restrictions"/>
+                        <br>
+                        <label class="flex items-center">
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.ambiant_music"/>
+                        <span class="ml-2 text-sm text-black">Ambiance Music</span>    
+                        </label> 
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.sale_for_profit"/>
```

Comment:
```suggestion
                        <label class="flex items-center">
                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.sale_for_profit"/>
```

Its missing a opening label tag which results in an incorrect indent:
![image](https://user-images.githubusercontent.com/21128579/105226167-a3852500-5b2d-11eb-8fc6-81d9fecb16ea.png)
I provided the fix to apply.

Reply:
Thank you.